cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -123,33 +145,33 @@ if(is_mpi)
     if(NOT MPI_CXX_FOUND)
       # Fix concurrent output to console
       execute_process(
-          COMMAND
```

Comment:
Doh! Sorry about that, fixing....

On 1 June 2015 at 18:49, Ruslan Baratov <notifications@github.com> wrote:

> In cmake/schemes/url_sha1_boost_library.cmake.in
> <https://github.com/ruslo/hunter/pull/159#discussion_r31479014>:
>
> > @@ -123,33 +145,33 @@ if(is_mpi)
> >      if(NOT MPI_CXX_FOUND)
> >        # Fix concurrent output to console
> >        execute_process(
> > -          COMMAND
>
> Actually this indentation is by design. The rule is 2 for blocks (e.g. if
> or foreach):
>
> if(A)
>   # 2 spaces
>   if(B)
>     # + 2 spaces
>     foreach(x ${y})
>       # + 2 spaces
>       message("x = ${x}")
>     endforeach()
>   endif()endif()
>
> and 4 spaces for arguments:
>
> configure_file("${x}" "${y}" @ONLY) # short enough
> # longerconfigure_file(
>     # 4 spaces
>     "/path/to/x" "/path/to/y" @ONLY
> )
> # very longconfigure_file(
>     # 4 spaces
>     "/very/long/path/to/x"
>     "/very/long/path/to/y"
>     @ONLY
> )
>
> Mix:
>
> if(A)
>   # +2 for block
>   execute_process(
>       # +4 for arguments
>       COMMAND "${CMAKE_COMMAND}" -E echo "Hello"
>       WORKING_DIRECTORY "/path/to/working/directory"
>   )
>   message("DONE")
>   if(B)
>     # +2 for block
>     message("With B")
>   endif()endif()
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ruslo/hunter/pull/159/files#r31479014>.
>



-- 
Alexandre Pretyman